app/views/layouts/application.html.erb

```diff
@@ -37,6 +37,9 @@
             <% end %>
           </div>
 
+          <%= breadcrumbs display_single_fragment: true, style: :bootstrap5 %>
```

Comment:
Could this go in a card or something to help it stand out a little more? Or, wild idea, stick it in the top bar maybe?